sdk/python/feast/cli.py

```diff
@@ -353,5 +356,38 @@ def project_list():
     print(tabulate(table, headers=["NAME"], tablefmt="plain"))
 
 
+@cli.command("apply")
+@click.argument("repo_path", type=click.Path(dir_okay=True, exists=True))
+def apply_total_command(repo_path: str):
+    """
+    Applies a feature repo
+    """
+    repo_config = load_repo_config(Path(repo_path))
```

Comment:
The plan is to eventually use the FeatureStore class here right?

Reply:
Yeah I imagine there will be a nicer interface for notebooks. I just tend to implement bits of functionality as pure functions by default (where possible), so they're easier to unit test.

Reply:
Yea I agree with that approach, I just want to make sure that we dont have two code paths that function differently.